cmd/fluent-bit/Dockerfile

```diff
@@ -3,7 +3,7 @@ COPY . /src/loki
 WORKDIR /src/loki
 RUN make clean && make BUILD_IN_CONTAINER=false fluent-bit-plugin
 
-FROM fluent/fluent-bit:1.2
+FROM fluent/fluent-bit:1.4
```

Comment:
From the [FLB docu](https://docs.fluentbit.io/manual/installation/upgrade-notes) there are not breaking changes